bricks/diagram/src/draw-canvas/decorators/DecoratorText.tsx

```diff
@@ -38,6 +38,13 @@ export function DecoratorText({
       element.textContent = currentLabel;
     }
   }, [currentLabel]);
+  useEffect(() => {
+    const element = ref.current;
```

Comment:
应该用它的父元素的尺寸，因为边框是加在它上面的

```suggestion
    const element = ref.current?.parentElement;
```

Reply:
一开始我是加父边框的，但是仔细想一下感觉直接文字容器的大小就可以了吧，不然实际操作需要框选的范围可能比用户预期的要大的多

Reply:
那你拿它的 x/y 也要减去容器的 padding 值